shared/common-adapters/clickable-box.js.flow

```diff
@@ -14,6 +14,11 @@ export type Props = {|
   feedback?: boolean,
   // mobile only
   activeOpacity?: number,
+  // desktop only
```

Comment:
Those calls need to be forwarded like this, otherwise it doesn't work, I just tried it again. Marco found this when I pestered him for some help.